USEPA/EPA-Cyano-Web
# Cyan 537 backend reqs

Update script for Python backend requirements for cyan_flask. This PR also includes #99, which fixes the unit and integration tests after upgrading pyjwt library.

To test:
1. Rebuild cyan-api service with latest code.
2. Hop into the cyan-api container.
3. Run `sh /docker/flask/update-packages.sh`
4. `EPA-Cyano-Web/docker/flask/requirements.new` file gets created on host and container.
5. Copy requirements from said file to `EPA-Cyano-Web/cyan_flask/requirements.txt` (should already be up to date though).
